app/packages/looker/src/overlays/heatmap.ts

```diff
@@ -199,14 +199,18 @@ export default class HeatmapOverlay<State extends BaseState>
   private getColor(state: Readonly<State>, value: number): number {
```

Comment:
I'm not quite sure where this version of `getColor` is used, but I added clamp here too anyways.